prime-router/docs/api/check.yml

```diff
@@ -2,7 +2,7 @@ openapi: "3.0.3"
 info:
   title: ReportStream Check API
   description: "This is a stub to be completed when this API is reviewed"
```

Comment:
Please update the description:

Reply:
Heh. Good catch. Fun fact, this string appears to be copy pasta in 5x files total.

I updated #6243 to fix the other locations at some point.